src/middlewared/middlewared/plugins/chart_releases_linux/validation.py

```diff
@@ -136,3 +163,36 @@ async def validate_container_image(self, verrors, value, question, schema_name,
         else:
             if not digest:
                 verrors.add(schema_name, f'Unable to retrieve {tag!r} container image tag details.')
+
+    @private
+    async def validate_custom_host_path(self, verrors, path, question, schema_name, release_data):
```

Comment:
We should probably have this method as synchronous because of how it interacts with the filesystem